### ISSUE TITLE
Task file validation should use error markers

### ISSUE THREAD

**Issue by aeschli** [Microsoft/vscode#27727]
Testing: #27443

- follow steps as described in #27443
- customize the gulp watch task
- in `tasks.json` add a problem matcher: `"problemMatcher": "dd"`
- error appears in the 'Tasks' output.

Would be nicer to get that error in the tasks.json.
Having the output show messages while typing is quite surprising.

**Comment by dbaeumer** [comment]
Agree, but I think this needs a tasks.json language server since the problem matchers are an open set now. 

@aeschli can the schema help me here?

**Comment by dbaeumer** [comment]
What is in for June is that the schema validates them but they appear as green squiggles.

@aeschli can I control in the schema if something is a warning or an error. That would be cool.

**Comment by aeschli** [comment]
@dbaeumer Sorry, currently its all warnings. Please file a request.

**Comment by dbaeumer** [comment]
@aeschli  here you go https://github.com/Microsoft/vscode/issues/29527

**Comment by alexr00** [comment]
Linked issue is out of scope. Therefore this one is too.
